aot/src/test/java/com/dylibso/chicory/experimental/aot/ClassTooLargeTest.java

```diff
@@ -6,20 +6,20 @@
 import com.dylibso.chicory.runtime.Instance;
 import com.dylibso.chicory.wabt.Wat2Wasm;
 import com.dylibso.chicory.wasm.Parser;
-import com.github.jknack.handlebars.Handlebars;
-import com.github.jknack.handlebars.helper.ConditionalHelpers;
 import java.io.IOException;
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
+import java.io.StringWriter;
 import java.util.ArrayList;
-import java.util.List;
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.apache.velocity.runtime.RuntimeConstants;
+import org.apache.velocity.runtime.resource.loader.ClasspathResourceLoader;
 import org.junit.jupiter.api.Test;
 
-public class ClassTooLarge {
+public class ClassTooLargeTest {
```

Comment:
this rename is important, alternatively the test won't be executed automatically ...